nanover-rs/src/tracked_simulation/openmm.rs

```diff
@@ -246,26 +234,3 @@ fn add_force_map_to_frame(force_map: &CoordMap, frame: &mut FrameData) {
         .insert_float_array("forces.user.sparse", sparse)
         .unwrap();
 }
-
-// Currently, this function has to retrieve the positions of the atoms with which the user is
```

Comment:
This function may be useful for calculating the work done on the system, I don't know if we want to remove it at this stage? But maybe it would be better to just rewrite it when it is relevant rather than keep it hanging around.

Reply:
In general it's better to avoid keeping code around that isn't being used--it can always be recovered from the repository history. I wouldn't have a problem keeping it if you prefer though.

Reply:
No I agree with you, I can just grab it from a previous PR if it ends up being needed! Get it out of there